code/game/objects/items/weapons/mop.dm

```diff
@@ -21,14 +21,19 @@
 	var/cleantime = 25
 	var/last_clean
 	var/clean_msg = FALSE
+	// If it's Horizon janitorial supplies, track it for the app.
+	var/tracked_supply = 0
```

Comment:
if this is a bool it should be `tracked_supply = FALSE` instead. ditto for all other instances as well.